sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -534,7 +534,7 @@ Table 7. Member functions of the `command_graph` class (memory operations).
 using namespace ext::oneapi::experimental;
 node add_malloc_device(void *&data, size_t numBytes, const std::vector<node>& dep = {});
 ----
-|Adding a node that encapsulates a memory allocation operation.
+|Adds a node that encapsulates a USM device memory allocation operation.
```

Comment:
My understanding is that this is equivalent to a USM device allocation, saying this explicitly means that a reading can just consult that section of the SYCL spec for valid usage of the pointer returned by `data`. However if `add_malloc_device` should be used differently, we should specify that expectation here instead.

Reply:
I think the usage of the pointers returned should be the same as a regular USM allocation, we could note that specifically here I suppose just to be extra clear.